### ISSUE TITLE
Runtime downloads are broken

### ISSUE THREAD

**Issue by MattKiazyk** [XcodesOrg/xcodes#299]
xcodes runtimes

> Error: dataCorrupted(Swift.DecodingError.Context(codingPath: [CodingKeys(stringValue: "sdkToSeedMappings", intValue: nil), _PlistKey(stringValue: "Index 51", intValue: 51), CodingKeys(stringValue: "platform", intValue: nil)], debugDescription: "Cannot initialize Platform from invalid String value com.apple.platform.xros", underlyingError: nil))

ie: support `com.apple.platform.xros` 😱

**Comment by davdroman** [comment]
Same here. I'm afraid our CI is blocked by this too 😕 https://github.com/siteline/SwiftUI-Introspect/pull/261

**Comment by cmdrsabre** [comment]
closed? So how do I fix / get rid of this?

**Comment by MattKiazyk** [comment]
@cmdrsabre this issue was around not being able to do anything because of the addition of VisionOS runtime.

If you are using the latest, things should be working. If not, please add another ticket
